OasisDEX/oasis-borrow
# 🧿 Ajna earn simulation

# [Ajna earn simulation](https://app.shortcut.com/oazo-apps/story/7857/ajna-earn-simulation-front-end)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added handling for simulation when opening Ajna earn position
- replaced hardcoded values in order information and ajna manage oveview
- created graphql query to read data about pool
  
## How to test 🧪
  <Please explain how to test your changes>

- try to open / manage ajna position, verify whether numbers on UI makes sense

## Things to do in the next step

- earnings
- rewards
- handling data in pool selector table
- simulation section skeleton loading
- when it will be possible to open earn position without nft we will need to provide logic to handle between gas estimations
